web/components/issues/issue-modal/form.tsx

```diff
@@ -304,7 +305,9 @@ export const IssueFormRoot: FC<IssueFormProps> = observer((props) => {
                         handleFormChange();
                       }}
                       buttonVariant="border-with-text"
-                      // TODO: update tabIndex logic
+                      renderCondition={(project) =>
+                        !!project.member_role && project.member_role >= EUserProjectRoles.MEMBER
```

Comment:
This seems to be repeated, se if you can write a util class for this which takes the role and returns boolean